typescript/graphql-nextjs/package.json

```diff
@@ -33,5 +34,10 @@
   },
   "prisma": {
     "seed": "ts-node prisma/seed.ts"
+  },
+  "pkg": {
+    "assets": [
+      "node_modules/.prisma/client/*.node"
+    ]
```

Comment:
Could you clarify what this snippet does & how it impacts this example?

Reply:
Found the solution here - https://www.prisma.io/docs/guides/other/troubleshooting-orm/help-articles/pkg-issue

Reply:
Anyone actively maintains this repo?

I can volunteer!